gridstatus/tests/test_ercot.py

```diff
@@ -74,6 +74,18 @@ def test_ercot_get_load_today():
     assert df["Time"].unique()[0].date() == today
 
 
+def test_ercot_get_load_latest():
+    expected_keys = {
+        "Time",
+        "Load",
```

Comment:
this is a weirdness in the current gridstatus api, but `get_load("latest")` returns a dictionary with all lower case keys for all the ISOs. 

this is why `test_get_latest_load` in `test_isos.py` is failing after this change

Reply:
Ahh my bad.

Reply:
Fixed this. I misread the test failure as an unrelated issue.

However, this latest failure is unrelated, I believe: https://github.com/kmax12/gridstatus/actions/runs/3790437409/jobs/6445062172

Reply:
ya, i agree the latest failure all look unrelated. tests seem to really be running slow today 😢